src/main/java/org/quantumbadger/redreader/activities/BaseActivity.java

```diff
@@ -79,6 +84,18 @@ else if (orientation == PrefsUtility.ScreenOrientation.LANDSCAPE)
 			setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
 	}
 
+	private void setTorFromPrefs(boolean force) {
+		tor = PrefsUtility.network_tor(this, sharedPreferences);
+		if (tor) {
+			if (OrbotHelper.isOrbotInstalled(this)){
+				OrbotHelper.requestStartTor(this);
+			} else {
+				OrbotHelper.getOrbotInstallIntent(this);
```

Comment:
Does this line do anything? AFAICS getOrbotInstallIntent just returns an intent, and doesn't start the activity by itself.